test/unit/connection-manager.test.js

```diff
@@ -1,104 +0,0 @@
-'use strict';
-
-const chai = require('chai');
-const sinon = require('sinon');
-
-const expect = chai.expect;
-const Support = require('./support');
-const { ConnectionManager } = require('@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/connection-manager.js');
-
-describe('connection manager', () => {
-  describe('_connect', () => {
-    beforeEach(function () {
-      this.connection = {};
-
-      this.dialect = {
-        connectionManager: {
-          connect: sinon.stub().resolves(this.connection),
-        },
-      };
-
-      this.sequelize = Support.createSequelizeInstance();
-    });
-
-    it('should resolve connection on dialect connection manager', async function () {
```

Comment:
This behavior has been removed because it doesn't really make sense. It checks that `ConnectionManager._connect` calls `connect` not on *itself* but on `dialect.connectionManager.connect`.

I think this is a legacy test from before Sequelize moved to using native classes